docs/DeveloperGuide.md

```diff
@@ -426,6 +426,12 @@ ShortcutSettings implements the `Serializable` interface, thus is saved to `json
     * Pros: Will use less memory (No extra data structure created).
     * Cons: Difficult to manage duplicate shortcut mappings.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## **Planned Enhancements**
+
+1. DoConnek Pro currently does not check for duplicate persons. This means that people with duplicate parameters (`Name`, `Phone`, `Email` etc.) can be added. 
+We plan on implementing an `NRIC` field as a unique identifier to solve this issue. 
```

Comment:
Can I clarify if `NRIC` will be an identifier for both patient and specialists? Because getting `NRIC` for specialists might be quite unconventional

Reply:
Valid point...I guess we can still use `NRIC` to identify patients but specialists could be some other (more extensive than just name) check on the parameters? Interested to hear the others' thoughts on this issue also.

Reply:
Instead of NRIC, we could instead uniquely identify Specialists with MCR Number. A quick google search resulted in:

> The MCR number for doctors in Singapore is the Medical Registration Number issued by the Singapore Medical Council (SMC). It is a unique identification number that is assigned to all doctors who are registered with the SMC. The MCR number is used to verify the credentials of doctors in Singapore and to ensure that they are qualified to practice medicine.

![image](https://github.com/AY2324S1-CS2103T-W13-1/tp/assets/95404279/b33cdb86-23c6-4a4d-b654-011f540ed1b0)
Taken from: 
https://www.moh.gov.sg/hpp/all-healthcare-professionals/healthcare-professionals-search

Reply:
Great, will update the DG to reflect this.